lib/elastic_searcher.rb

```diff
@@ -59,6 +58,9 @@ def search_definition # rubocop:disable Metrics/MethodLength
         end
       end
 
+      # only return gems that are not yanked unless filter is used
+      post_filter { term yanked: false } unless query_str.include? "yanked:true"
```

Comment:
is there a way we don't have to hardcode `yanked:true`? filter can also be written as `yanked: true`. ruby-es may have option for parsed query.